README.md

```diff
@@ -44,6 +44,14 @@ kafka.metrics.reporters=io.strimzi.kafka.metrics.prometheus.ServerYammerMetricsR
 auto.include.jmx.reporter=false
 ```
 
+The `prometheus.metrics.reporter.allowlist` configuration of brokers and controllers can be updated at runtime using the `kafka-configs.sh` tool or the `incrementalAlterConfigs()` method from the `Admin` API. For example:
+
+```sh
+./bin/kafka-configs.sh --bootstrap-server localhost:9092 \
+  --alter --entity-type brokers --entity-default \
+  --add-config "prometheus.metrics.reporter.allowlist=[kafka_controller.*,kafka_log.*]"
+```
+
```

Comment:
Should we also include sample Java code demonstrating the `Admin` API usage?

Reply:
It might be useful?

Reply:
I've added an example